spec/models/profile_spec.rb

```diff
@@ -299,9 +299,39 @@
 
       expect(profile).to be_active
     end
+
+    context 'when the initiating_sp is the IRS' do
+      it 'logs an attempt event' do
+        sp = create(:service_provider, :irs)
+        profile = create(
+          :profile, user: user, active: false, fraud_review_pending: true,
+                    initiating_service_provider: sp
+        )
+        expect(profile.initiating_service_provider.irs_attempts_api_enabled?).to be_truthy
+
+        expect_any_instance_of(IrsAttemptsApi::Tracker).to receive(:fraud_review_adjudicated).
+          with(determination: 'activated')
+        profile.activate_after_passing_review
+      end
+    end
+
+    context 'when the initiating_sp is not the IRS' do
+      it 'does not log an attempt event' do
+        sp = create(:service_provider)
+        profile = create(
+          :profile, user: user, active: false, fraud_review_pending: true,
+                    initiating_service_provider: sp
+        )
+        expect(profile.initiating_service_provider.irs_attempts_api_enabled?).to be_falsey
```

Comment:
I'm using this and 310 just to highlight that "Are they the IRS?" is actually hinging on this parameter, which is what the code uses. This are otherwise dumb test assertions just testing the factory itself.